utilities/test_suite/HOST/testAllScript.sh

```diff
@@ -4,7 +4,7 @@ cwd=$(pwd)
 
```

Comment:
are the csv files supposed to be checked in as well?

Reply:
The CSV basically contains the golden ouptut for small 50x50/100x100/150x150 images to pass QA tests